### PR TITLE
Cfspdk 1202/1203 DFU library

### PR DESCRIPTION
**Pull request by YHusain1**
Moved the Murata and WiFi DFU source code to a new `libs` location.

### CODE REVIEW

File: libs/dfu_common.h
Comment (jvhutch) [inline libs/dfu_common.h:1]: Yusuf, let's move this file to the dfu_gecko subdirectory.

**Comment by jvhutch** [comment]
Yusuf, let's add some c-language macro enhancements to Tmo Shell such that those that follow us can disable the inclusion of the DFU Gecko Library, e.g., ".../tmo_shell/prj.con: CONFIG_DFU_GECKO_LIB=y" can be set to 'n'. I've tested the feasibility of this suggestion, and I have a git patch I will convey via Slack. Note, that this enhancement will make it easier for David's team to understand how to use the library.

File: libs/dfu_common.h
Comment (johnlange2) [reply libs/dfu_common.h:1]: would prefer not to move this file

**Review by jtbaumann — APPROVED** [review]

**Review by johnlange2 — APPROVED** [review]

File: libs/dfu_common.h
Comment (jvhutch) [reply libs/dfu_common.h:1]: As we discussed earlier, I agree; we'll make a better schema in the near future.

**Review by jvhutch — APPROVED** [review]

**Review by johnlange2 — APPROVED** [review]

**Pull request merged**